### PR TITLE
[ENG-237] Add check for existing env file

### PR DESCRIPTION
**Pull request by cheran-senthil**

### CODE REVIEW

File: content-node/docker-compose/setup.sh [hunk 1]
```diff
@@ -141,6 +141,23 @@ get_validated_input() {
     done
 }
 
+# Check if .env file exists and prompt for confirmation
+if [ -f ".env" ]; then
+    log_warning "An .env file already exists!"
+    read -p "Do you want to overwrite it? (y/n): " overwrite
```
Comment (dmanjunath) [inline content-node/docker-compose/setup.sh:147]: can we make the text here like "Would you like to go through the .env setup process again?"

File: content-node/docker-compose/setup.sh [hunk 2]
```diff
@@ -141,6 +141,23 @@ get_validated_input() {
...
+        log_info "Keeping existing .env file."
+        echo
+        log_info "To start the content node, use: ${GREEN}docker compose up -d${NC}"
+        # Remind to restart if docker group was added
+        if groups $USER | grep -q "\bdocker\b"; then
+            log_warning "Please log out and log back in for Docker group changes to take effect."
+        fi
+        exit 0
+    fi
+    log_info "Proceeding to overwrite existing .env file..."
```
Comment (dmanjunath) [inline content-node/docker-compose/setup.sh:158]: do you need to delete existing file here?

File: content-node/docker-compose/setup.sh [hunk 3]
```diff
@@ -141,6 +141,23 @@ get_validated_input() {
     done
 }
 
+# Check if .env file exists and prompt for confirmation
+if [ -f ".env" ]; then
+    log_warning "An .env file already exists!"
```
Comment (dmanjunath) [inline content-node/docker-compose/setup.sh:146]: should we print values here?

**Review by dmanjunath — APPROVED** [review]

File: content-node/docker-compose/setup.sh (hunk 3 above)
Comment (cheran-senthil) [reply content-node/docker-compose/setup.sh:146]: Yeah, there are no secrets, we can log this.

File: content-node/docker-compose/setup.sh (hunk 2 above)
Comment (cheran-senthil) [reply content-node/docker-compose/setup.sh:158]: We can rename this old file 🤔

**Pull request merged**